src/ray/raylet/scheduling/local_resource_manager.cc

```diff
@@ -563,4 +572,9 @@ void LocalResourceManager::RecordMetrics() const {
   }
 }
 
+void LocalResourceManager::SetLocalNodeDraining() {
+  is_local_node_draining_ = true;
+  OnResourceChanged();
```

Comment:
can you comment why this is necessary? (it is unintuitive since is_local_node_draining_ != resource change?)